apps/src/code-studio/pd/teacher_application/teacher_application.jsx

```diff
@@ -133,6 +133,12 @@ const TeacherApplication = React.createClass({
   },
 
   getInitialState() {
+    if (this.props.accountEmail) {
```

Comment:
You could just do 
`return this.props.accountEmail ? {primaryEmail: This.props.accountEmail} : {}`